docs/version

```diff
@@ -1,4 +1,4 @@
 #! /bin/bash
 
-VERSION="0.2.0"
+VERSION="v0.3.0"
```

Comment:
To be consistent with previous versions, just use the version number without `v`

Reply:
develop points to "v0.2.0", but that was never propagated to master. GitHub releases are also labelled as vX.Y.Z. Should we keep the "v" or ditch it?

Reply:
you're absolutely right!! Lets keep the "v" :)